chain-spec-generator/src/system_parachains_specs.rs

```diff
@@ -758,3 +769,77 @@ pub fn people_kusama_local_testnet_config() -> Result<Box<dyn ChainSpec>, String
 		.build(),
 	))
 }
+
+// PeoplePolkadot
+fn people_polkadot_genesis(
+	invulnerables: Vec<(AccountId, AuraId)>,
+	endowed_accounts: Vec<AccountId>,
+	id: ParaId,
+) -> serde_json::Value {
+	serde_json::json!({
+		"balances": people_polkadot_runtime::BalancesConfig {
+			balances: endowed_accounts
+				.iter()
+				.cloned()
+				.map(|k| (k, PEOPLE_POLKADOT_ED * 4096 * 4096))
```

Comment:
Just a magic number or does the 2^24 have a meaning?

Reply:
Magic number AFAIK that we've used for all local setups